yarn.lock

```diff
@@ -2358,7 +2358,7 @@
     "@babel/runtime" "^7.20.1"
     prop-types "^15.8.1"
 
-"@mui/system@5.11.12", "@mui/system@^5.11.12":
+"@mui/system@^5.11.12":
```

Comment:
I think if we redo yarn.lock from scratch we shouldn't see mui/system in here at all

Reply:
I think `@mui/material` depends on it 👇

<img width="1157" alt="Screenshot 2023-03-21 at 10 40 11" src="https://user-images.githubusercontent.com/29652890/226568103-665716e3-58fc-48b2-a683-df84ac04a833.png">